tasks.py

```diff
@@ -268,3 +268,29 @@ def check_pypi_version(context, name=PROJECT_NAME, version=PROJECT_VERSION):
     print(f"The version {version} does not exist on PyPI.")
     print("The version can be released.")
     sys.exit(0)
+
+
+@task
+def html(context, sourcedir="docs/source", builddir="docs/build"):
+    """Creates html docs using sphinx-build command.
+
+    Args:
+        context (obj): Used to run specific commands
+        sourcedir (str, optional): Source directory for sphinx to use. Defaults to "source".
+        builddir (str, optional): Output directory for sphinx to use. Defaults to "build".
+    """
+    print("Building html documentation...")
```

Comment:
I think it makes sense to call clean_docs here, just ensure it is done idempotently.

Reply:
Updated